### PR TITLE
feat: file transfer from reroils-data

### PR DESCRIPTION
**Pull request by rerowep**
Signed-off-by: Peter Weber <Peter.Weber@rero.ch>
Co-authored-by: Bertrand Zuchuat <bertrand.zuchuat@rero.ch>

Verify: 
all nedded files from https://github.com/rero/reroils-data-legacy/tree/auth-master have been transferred 

Test:
./scripts/bootstrap
./run_tests.ch

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -42,6 +42,7 @@ htmlcov/
 .coverage
 .coverage.*
 .cache
+.pytest_cache
```
Comment (jma) [inline .gitignore]: Not sure

File: MANIFEST.in [hunk 2]
```diff
@@ -56,11 +55,13 @@ include LICENSE
 include Pipfile
 include babel.ini
 include pytest.ini
+recursive-include data *.json
 recursive-include docs *.bat
 recursive-include docs *.py
 recursive-include docs *.rst
 recursive-include docs *.txt
 recursive-include docs Makefile
-recursive-include rero_mef *.gitkeep
```
Comment (jma) [inline MANIFEST.in]: Why?

File: rero_mef/authorities/marctojson/do_bnf_auth_person.py [hunk 3]
```diff
@@ -0,0 +1,218 @@
...
+__author__ = "Gianni Pante <Gianni Pante@rero.ch>"
+__version__ = "0.0.1"
+__copyright__ = "Copyright (c) 2009-2018 Rero, Gianni Pante"
+__license__ = "Internal Use Only"
+
+
+# ----------------------------------- Classes ---------------------------------
+
+# MrcIterator ----
+class Transformation(object):
```
Comment (jma) [inline rero_mef/authorities/marctojson/do_bnf_auth_person.py]: 1. BnfTransformation instead? If yes do the same for the others.
2. Create a base class for the common methods.

File: rero_mef/authorities/marctojson/do_bnf_auth_person.py [hunk 4]
```diff
@@ -0,0 +1,218 @@
...
+    """Transformation unimarc to json for bnf autority person."""
+
+    def __init__(self, marc, logger=None, verbose=False, transform=True):
+        """Constructor."""
+        self.marc = marc
+        self.logger = logger
+        self.verbose = verbose
+        self.json_dict = {}
+        if transform:
```
Comment (jma) [inline rero_mef/authorities/marctojson/do_bnf_auth_person.py]: Why? More generally we can have a class method such as: set_marc(marc) that do the transformation and a cached property for `json`.

File: rero_mef/authorities/marctojson/do_skeleton.py [hunk 5]
```diff
@@ -0,0 +1,91 @@
...
+
+__author__ = "Johnny Mariethoz <Johnny.Mariethoz@rero.ch>"
+__version__ = "0.0.1"
+__copyright__ = "Copyright (c) 2009-2011 Rero, Johnny Mariethoz"
+__license__ = "Internal Use Only"
+
+
+# ----------------------------------- Classes ---------------------------------
+# MrcIterator ----
+class Transformation(object):
```
Comment (jma) [inline rero_mef/authorities/marctojson/do_skeleton.py]: Can be a base class?

File: rero_mef/authorities/marctojson/helper.py [hunk 6]
```diff
@@ -0,0 +1,176 @@
...
+    """Make nice marc record."""
+    nice = ''
+    for field in record:
+        nice += '%s: %s' % (field.tag, nice_marc_field(field, ctrl)) + '\n'
+    return nice
+
+
+def display_record(record, ctrl=False):
+    """Displays the record pymarc like."""
+    print(nice_record(record, ctrl))
```
Comment (jma) [inline rero_mef/authorities/marctojson/helper.py]: click?

File: rero_mef/authorities/marctojson/logger.py [hunk 7]
```diff
@@ -0,0 +1,145 @@
...
+    """Base class for errors in the Logger packages."""
+
+    class InvalidileName(Exception):
+        """The given file name is not correct."""
+
+        pass
+
+
+# Logger ---
+class Logger:
```
Comment (jma) [inline rero_mef/authorities/marctojson/logger.py]: Why not `invenio-logging`?

File: rero_mef/authorities/marctojson/marctojson.py [hunk 8]
```diff
@@ -0,0 +1,152 @@
+#!/usr/bin/env python
```
Comment (jma) [inline rero_mef/authorities/marctojson/marctojson.py]: Can be a click script of better a dojson entrypoint.

File: rero_mef/authorities/marctojson/records.py [hunk 9]
```diff
@@ -0,0 +1,150 @@
+# -*- coding: utf-8 -*-
```
Comment (jma) [inline rero_mef/authorities/marctojson/records.py]: Can be replaced by dojson

File: rero_mef/authorities/providers.py [hunk 10]
```diff
@@ -0,0 +1,219 @@
+# -*- coding: utf-8 -*-
```
Comment (jma) [inline rero_mef/authorities/providers.py]: Provider are useless, take the source id as PID!

File: tests/authorities/test_bnf_auth_person.py [hunk 11]
```diff
@@ -0,0 +1,331 @@
...
+
+def test_bnf_gender_missing():
+    """Test gender 120 missing"""
+    xml_part_to_add = ""
+    trans = trans_prep('bnf', xml_part_to_add)
+    trans.trans_bnf_gender()
+    assert trans.json == {}
+
+
+def test_bnf_language_of_person_1():
```
Comment (jma) [inline tests/authorities/test_bnf_auth_person.py]: _1 is a bad name

File: tests/authorities/test_bnf_auth_person.py [hunk 12]
```diff
@@ -0,0 +1,331 @@
...
+    """Test language of person 101 $a (langue sur 3 caractères) ISO 639-2"""
```
Comment (jma) [inline tests/authorities/test_bnf_auth_person.py]: comment in french

File: tests/authorities/test_bnf_auth_person.py [hunk 13]
```diff
@@ -0,0 +1,331 @@
...
+    trans.trans_bnf_language_of_person()
+    assert trans.json == {
+        "language_of_person": [
+            "fre",
+            "eng"
+        ]
+    }
+
+
+def test_bnf_language_of_perso_2():
```
Comment (jma) [inline tests/authorities/test_bnf_auth_person.py]: bad name

File: tests/authorities/test_gnd_auth_person.py [hunk 14]
```diff
@@ -0,0 +1,329 @@
...
+
+def test_gnd_gender_missing():
+    """Test gender 375 missing"""
+    xml_part_to_add = ""
+    trans = trans_prep('gnd', xml_part_to_add)
+    trans.trans_gnd_gender()
+    assert trans.json == {}
+
+
+def test_gnd_language_of_person_1():
```
Comment (jma) [inline tests/authorities/test_gnd_auth_person.py]: bad name

File: tests/authorities/test_gnd_auth_person.py [hunk 15]
```diff
@@ -0,0 +1,329 @@
...
+    """Test language of person 377 $a (langue sur 3 caractères) ISO 639-2b"""
```
Comment (jma) [inline tests/authorities/test_gnd_auth_person.py]: in french (copy/paste)

File: tests/authorities/test_gnd_auth_person.py [hunk 16]
```diff
@@ -0,0 +1,329 @@
...
+    trans.trans_gnd_language_of_person()
+    assert trans.json == {
+        "language_of_person": [
+            "fre",
+            "eng"
+        ]
+    }
+
+
+def test_gnd_language_of_person_2():
```
Comment (jma) [inline tests/authorities/test_gnd_auth_person.py]: bad name

File: tests/ui/test_views.py [hunk 17]
```diff
@@ -24,13 +24,13 @@
 
 """Views tests."""
 
-from __future__ import absolute_import, print_function
```
Comment (jma) [inline tests/ui/test_views.py]: Why?

Comment (rerowep) [reply tests/ui/test_views.py]: It's from the cookiecutter, You have to ask  invenio.

File: tests/authorities/test_gnd_auth_person.py (hunk 16 above)
Comment (rerowep) [reply tests/authorities/test_gnd_auth_person.py]: Comes from baa et pag

File: tests/authorities/test_gnd_auth_person.py (hunk 15 above)
Comment (rerowep) [reply tests/authorities/test_gnd_auth_person.py]: Comes from baa et pag

File: tests/authorities/test_gnd_auth_person.py (hunk 14 above)
Comment (rerowep) [reply tests/authorities/test_gnd_auth_person.py]: Comes from baa et pag

File: tests/authorities/test_bnf_auth_person.py (hunk 13 above)
Comment (rerowep) [reply tests/authorities/test_bnf_auth_person.py]: Comes from baa et pag

File: .gitignore (hunk 1 above)
Comment (rerowep) [reply .gitignore]: It was always on `git status`

File: MANIFEST.in (hunk 2 above)
Comment (rerowep) [reply MANIFEST.in]: Comes from `check-manifest -u`

**Pull request merged**